### DISCUSSION TITLE
Can't start my codespace

### DISCUSSION THREAD

**Discussion by phidipedx** [community/community#45631] [Codespaces]
### Select Topic Area

Bug

### Body

I am following cs50x and have been working on a codespace throughout the course. Today it does not start. I get a

Oh no, something unexpected went wrong
-- 500 internal server error

When I try to restart, the circle indicating it's running just keep spinning forever.

I have tried logging out, shutting the browser, and restartied everything to no avail. Please advice

/phidipedx

**Comment by jge162** [comment]
it is difficult to say exactly what is causing the issue, you need to contact the [ Codespace's support team](https://support.github.com/request/codespaces?tags=hubberfy_codespaces)

**Comment by GideonCheruiyot** [comment]
Hey @phidipedx, is this still happening? what is the codespace url on the browser?

**Comment by phidipedx** [comment]
Hi Gideon

I was workin on it yesterday afternoon. I didn’t see the error message, but three or four times there were about 5 minute intervals where there was no contact to the webserver. It flashes a white background and then a black screen from the webserver.
I have a hunch that a button to create a container reset would help.

Cheers
Niels

**Comment by phidipedx** [comment]
Sorry, forgot, the Url flashes to my server, then back to the url with the port 5000. Think it is a flask init problem

**Comment by GideonCheruiyot** [comment]
Yea sounds like a port forwarding issue, please record a screen recording when you see it again, and if possible, some steps to reproduce so the team can work on a fix.

**Comment by phidipedx** [comment]
After some days with other activities, I am now trying to work on my project again.
I am facing the problem again:

from my codespace I execute 'flask run'
I am presented with a url that I ctrl-click
My application flickers a white screen, and then return to a black screen with the url https://auth.preview.app.github.dev/authenticate-codespace/phidipedx-code50-116666796-j6554ggvxxw3wr4?cid=aae89e8b691225b41eb3a3e9e60794df&port=5000&visibility=private

If I close the app with ctrl-c, close the browser tab and try again, the same thing happens.

I I close everything, and do a new login to the codespace via github, I may be lucky that it works. So that is what I do until it works.

Cheers
Niels

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------
